README.md

```diff
@@ -34,25 +36,46 @@ Open the source code and start editing!
 
 Note: The faucet is limited to one transfer per wallet every 6500 blocks (~1 day)
 
+
 **Rinkeby faucet for WETH:**
 [Wrap rinkeby eth on rinkeby uniswap](https://app.uniswap.org/#/swap)
 
 **Rinkeby faucets for LUSD, FRAX & DAI can be taken from rinkeby etherscan:**
 
 1. Go to `src/helpers/AllBonds.ts`
-2. then copy the rinkeby `reserveAddress` for the applicable bond & navigate to that contract on rinkeby etherscan. 
+2. then copy the rinkeby `reserveAddress` for the applicable bond & navigate to that contract on rinkeby etherscan.
 3. On Rinkeby etherscan use the `mint` function. You can use the number helper for 10^18 & then add four more zeros for 10,000 units of whichever reserve you are minting.
 
+## End to end test
+
+To run end to end tests you have to:
+
+- set up synpress to use the adequate network and wallet. For instance if you want to run tests on rinkeby network you could export
+
+```
+SECRET_WORDS=<your secret words>
+NETWORK_NAME=rinkeby
+```
+
+More options are available, please check: https://github.com/Synthetixio/synpress#-important
+
+- Make sure we have sufficent balance to run the tests
+
+- Actually run the tests
+  `yarn synpress:run`
+
+
```

Comment:
is this applicable to this PR? Or should it be removed until @JoeyR84 's PR for testing?
Or should we make it more clear right now that tests only apply to translations at this time?

Reply:
No Sry, Maybe a bad merge.
Removed.